app/concepts/matestack/ui/core/collection/filter/reset/reset.haml

```diff
@@ -0,0 +1,3 @@
+%a{@tag_attributes, "@click": "resetFilter()"}
```

Comment:
Same as submit, why not use a button element?

Reply:
because not everybody would like to use a button. using an `a` tag gives the matestack user more flexibility